services/cd-service/pkg/service/git_test.go

```diff
@@ -361,32 +421,68 @@ func TestGetCommitInfo(t *testing.T) {
 				CommitHash:    "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
 				CommitMessage: "some message",
 				TouchedApps: []string{
-					"app1",
-					"app2",
-					"app3",
+					"app-1",
```

Comment:
why do we need to rename this? and the environments too? (nitpick)

Reply:
I think it's needed because the environment setup is complex (multiple environment and environment groups). I also thought it's appropriate to change app names accordingly.